gtfs/spec/en/reference.md

```diff
@@ -505,6 +509,32 @@ Assigns stops from [stops.txt](#stopstxt) to areas.
 | `area_id` | Foreign ID referencing `areas.area_id` | **Required** | Identifies an area to which one or multiple `stop_id`s belong. The same `stop_id` may be defined in many `area_id`s. |
 | `stop_id` | Foreign ID referencing `stops.stop_id` | **Required** | Identifies a stop. If a station (i.e. a stop with `stops.location_type=1`) is defined in this field, it is assumed that all of its platforms (i.e. all stops with `stops.location_type=0` that have this station defined as `stops.parent_station`) are part of the same area. This behavior can be overridden by assigning platforms to other areas. |
 
+### networks.txt
+
+File: **Conditionally Forbidden**
+
+Primary key (`network_id`)
+
+Defines network identifiers that apply for fare leg rules. Forbidden if `routes.network_id` field exists.
+
+|  Field Name | Type | Presence | Description |
+|  ------ | ------ | ------ | ------ |
+| `network_id` | Unique ID | **Required** | Identifies a network. Must be unique in [networks.txt](#networkstxt). |
+| `network_name` | Text | **Optional** | The name of the network that apply for fare leg rules, as used by the local agency and its riders.
+
+### route_networks.txt
+
+File: **Conditionally Forbidden**
+
+Primary key (`*`)
```

Comment:
If a route_id can only be defined in one `network_id`, should the primary key be `route_id`?

Reply:
Yes, thanks for catching this! - https://github.com/google/transit/pull/405/commits/8664074722f24a373ce3fb0639a3d26798f4188d